ui/src/main/java/com/epam/eco/kafkamanager/ui/consumers/ConsumerGroupController.java

```diff
@@ -65,6 +67,7 @@
     public static final String MAPPING_GROUP = MAPPING_GROUPS + "/{name}";
     public static final String MAPPING_DELETE = MAPPING_GROUP + "/delete";
     public static final String MAPPING_GROUP_METADATA = MAPPING_GROUP + "/metadata";
+    public static final String MAPPING_GROUP_RESOURCE = MAPPING_GROUP + "/resource";
```

Comment:
I have set the url from resource permission page to resource page as:

/topics/{name}/resource?resourceType=TOPIC
/consumer_groups/{name}/resource?resourceType=GROUP
/transactions/{id}/resource?resourceType=TRANSACTIONAL_ID

Please let me know if the url needs to be changed.